spec/index.html

```diff
@@ -732,12 +754,15 @@ <h2>Literals</h2>
         not defined by this specification.</li>
     </ol>
 
-    <p><dfn data-local-lt="term-equal">Literal term equality</dfn>: Two literals are term-equal (the same
-      RDF literal) if and only if the two <a>lexical forms</a>,
-      the two <a>datatype IRIs</a>, and the two
-      <a>language tags</a> (if any) compare equal,
-      character by character. Thus, two literals can have the same value
-      without being the same RDF term. For example:</p>
+    <p><dfn data-local-lt="term-equal">Literal term equality</dfn>:
+      Two literals are term-equal (the same <a>RDF literal</a>)
+      if and only if the two <a>lexical forms</a>,
+      the two <a>datatype IRIs</a> compare equal, character by character,
+      the two <a>language tags</a> (if any) compare equal, as if normalized to lower case, and
+      the two <a>base directions</a> (if any) compare equal, character by character.
```

Comment:
the mention of "character to character" is odd, because currently, it seems to apply only to datatype IRI and direction, and not to lexical form nor language tag...

I propose the current change (which overrides my proposed change above for line 761)
```suggestion
      if and only if the two <a>lexical forms</a>,
      the two <a>datatype IRIs</a>,
      the two <a>language tags</a> (if any) normalized to lower case, and
      the two <a>base directions</a> (if any), compare equal, character by character.
```

It might be argued to be ambiguous (must they *all* be equal with each other?!) but that ambiguity existed in the text from RDF 1.1, and didn't seem to bother anyone :stuck_out_tongue:

Reply:
IIRc, I had something similar to this originally, but after feedback from @afs, spelled it out for each element.

Reply:
```suggestion
      if and only if the two <a>lexical forms</a>,
      the two <a>datatype IRIs</a>,
      the two <a>language tags</a> (if any) normalized to lower case, and
      the two <a>base directions</a> (if any),
      all compare equal, character by character.
```

Reply:
or
```suggestion
      if and only if all of
      the two <a>lexical forms</a>,
      the two <a>datatype IRIs</a>,
      the two <a>language tags</a> (if any) normalized to lower case, and
      the two <a>base directions</a> (if any),
      compare equal, character by character.
```

Reply:
There should probably be a colon after "if all of", as in "if and only  if all of:", as it's enumerating a set of clauses. I'm otherwise fine with this suggestion, but just be be clear, the language tags are normalized to lower case only for comparison purposes and the language tag elements, themselves, are not considered to necessarily be represented in lower case.